loadrunner_professional/manifest.json

```diff
@@ -0,0 +1,54 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "e6b5ab52-139d-4dde-a4ad-94fedeac7f29",
+  "app_id": "loadrunner-professional",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Send your LoadRunner Professional metrics from the scenario run, information about the scenario run, or both.",
+    "title": "LoadRunner Professional",
+    "media": [
```

Comment:
Please add at least one image here - we recommend starting with a screenshot of your dashboard, but you can also add any relevant screenshots of your platform or marketing media as well to give the user an idea of what their experience with the integration will look like.

Reply:
Like I said earlier, I cannot take a screenshot of the final version of the dashboard since the logo from the banner is not visible.
I added some screenshots of the product instead. And if possible, I will add a screenshot of the dashboard after the integration tile goes live.

Reply:
Thanks - that works for now